### ISSUE TITLE
add a new work/edition/book

### ISSUE THREAD

**Issue by bradyguyc** [Luca3317/OpenLibrary.NET#7]
have you seen away to add new book/work or edition through the api?   I saw something in their wiki stuff that made it sound like the only way to do that was through their internal server and that there is not a programmatic way to add books.

**Comment by Luca3317** [comment]
Nope, dont think they allow it since theyd just get flooded with bad data

**Comment by Luca3317** [comment]
As far as I can tell there isnt any.

**Comment by bradyguyc** [comment]
thanks much... I'll ask them on their Tuesday call..

**Comment by Luca3317** [comment]
Cool, lmk if it turns out there is

**Comment by bradyguyc** [comment]
I got this from the openlibrary.org team today.  He said on the call that we should be able to add works through this api.  Said that if we had problems with permissions to let them know and they would help out.  I am going to start looking at this this week.  Thanks

In terms of an import API, there's this: https://github.com/internetarchive/openlibrary/wiki/Developer's-Guide-to-Data-Importing#importing-json. I don't actually know what level of permissions that requires off the top of my head. The bot group may be enough. I think this is a good place for questions.

**Comment by Luca3317** [comment]
Interesting! I wont have time to work on this right away, but once I find the time Ill look into it.
If you get it working in the mean time, please share.

**Comment by bradyguyc** [comment]
your account has to be setup with a special permission to add books.  Thus would need to funnel all new book adds to a service that is using a special account.
